### PR TITLE
Move repository related configuration to GUI.

### PR DESCRIPTION
**Pull request by romangraef**

### CODE REVIEW

**Review by Lulonaut — APPROVED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/commands/repo/ResetRepoCommand.java [hunk 1]
```diff
@@ -1,18 +0,0 @@
...
-
-import io.github.moulberry.notenoughupdates.NotEnoughUpdates;
-import io.github.moulberry.notenoughupdates.commands.ClientCommandBase;
-import net.minecraft.command.CommandException;
-import net.minecraft.command.ICommandSender;
-
-public class ResetRepoCommand extends ClientCommandBase {
-
-	public ResetRepoCommand() {
-		super("neuresetrepo");
```
Comment (NopoTheGamer) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/repo/ResetRepoCommand.java:11]: can we not delete neuresetrepo?

**Review by NopoTheGamer — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/commands/repo/ResetRepoCommand.java (hunk 1 above)
Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/commands/repo/ResetRepoCommand.java:11]: It's pretty much useless

**Review by Lulonaut — DISMISSED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/commands/repo/ResetRepoCommand.java (hunk 1 above)
Comment (jani270) [reply src/main/java/io/github/moulberry/notenoughupdates/commands/repo/ResetRepoCommand.java:11]: You probably should change the message then.

File: src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java [hunk 2]
```diff
@@ -229,7 +229,6 @@ public CompletableFuture<Boolean> fetchRepository() {
 					return false;
 				}
 
-
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:241]: good change

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:241]: its your autoformat settings

File: src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/entityviewer/GUIClientPlayer.java [hunk 3]
```diff
@@ -33,7 +33,18 @@ public ResourceLocation getLocationCape() {
         }
     };
 
-    @Override
+	@Override
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/entityviewer/GUIClientPlayer.java:36]: I hate java

File: src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java [hunk 4]
```diff
@@ -1135,7 +1135,7 @@ public void consume(int x, int y, int id) {
 				}
 				if (internalname.get() != null) {
 					if (itemstack.get() != null) {
-						if (NotEnoughUpdates.INSTANCE.config.hidden.enableItemEditing && Keyboard.getEventCharacter() == 'k') {
+						if (NotEnoughUpdates.INSTANCE.config.apiKey.repositoryEditing && Keyboard.getEventCharacter() == 'k') {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1138]: why is it called apiKey here and the class is called api data??

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1138]: because otherwise everyone has to run /api new again

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1138]: i could write a bunch of code to change that, but i didnt feel like it

Comment (IRONM00N) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1138]: idc, make them run api new

Comment (jani270) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1138]: no

Comment (IRONM00N) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1138]: :angry:

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1138]: fine ill write the stupid code

**Review by IRONM00N — APPROVED** [review]
LGTM!!!

**Review by NopoTheGamer — APPROVED** [review]
:tf:

**Review by Lulonaut — APPROVED** [review]

**Pull request merged**